### DISCUSSION TITLE
Any way to import 'ROOT' as a symbol?

### DISCUSSION THREAD

**Discussion by danmysak** [explosion/spaCy#9020] [Help: Best practices]
It appears I can check whether a word is the subject of a sentence like this:

```
nlp('He knows')[0].dep == symbols.nsubj
```

However,

```
nlp('He knows')[1].dep == symbols.root
```

returns `False`, even though `nlp('He knows')[1].dep_` is `'ROOT'`. Moreover, `nlp('He knows')[1].dep` equals to `8206900633647566924`, a magic constant (or is it?) that I can’t find anywhere in the source code of the project, except in one of the tests where it is used as a literal.

My question is: is there a way to import the `'ROOT'` (or `8206900633647566924`) constant from spaCy, or the only option I have is to spell it out as a literal myself?

More generally, what is the reasoning behind asymmetric handling of the relation types such as `nsubj` or `amod` (which are included in `spacy.symbols`) and `ROOT` or `compound` (which are not)?

**Comment by adrianeboyd** [comment]
Which labels are in `symbols` is just related to historical reasons / backwards compatibility, and in spacy v2+ there's no particular reason to use them directly and I would recommend just referring to the string store. See more details in a related thread here: https://github.com/explosion/spaCy/discussions/8583#discussioncomment-956783

**Reply by danmysak** [reply]
Thank you!
I’d like to point out then that the snippet from https://spacy.io/usage/linguistic-features#navigating (the one with `from spacy.symbols import nsubj, VERB`) is a bit misleading :-)
